ph5/utilities/tests/test_ph5validate.py

```diff
@@ -0,0 +1,161 @@
+'''
+Tests for metadatatoph5
+'''
+import unittest
+import logging
+from StringIO import StringIO as StringBuffer
+import ph5
+from ph5.core import ph5api, experiment
+from ph5.utilities import ph5validate, texan2ph5, kef2ph5
+import os
+import shutil
+import tempfile
+
+
+class TestPh5Validate(unittest.TestCase):
+    def setUp(self):
```

Comment:
The setup and teardown should go in the general place.
If this is ready to go otherwise we can move it after